### ISSUE TITLE
205GAL and 208GAL - Duplicates?

### ISSUE THREAD

**Issue by geerlingguy** [ansible/ansible-lint#386]
> Note: This was copied over from the [galaxy-lint-rules] repository; please feel free to close if it doesn't currently apply to the existing or planned set of ansible-lint rules. Original issue: https://github.com/ansible/galaxy-lint-rules/issues/14

For the following line in one of my roles:

```yaml
    repo: 'deb https://oss-binaries.phusionpassenger.com/apt/passenger {{ansible_distribution_release}} main'
```

I got both of the following warnings:

```
[205GAL] Variables should be enclosed by spaces "{{ foo }}"
/Users/jeff.geerling/Dropbox/VMs/roles/geerlingguy.passenger/tasks/main.yml:23
    repo: 'deb https://oss-binaries.phusionpassenger.com/apt/passenger {{ansible_distribution_release}} main'

[208GAL] Variables should have spaces after {{ and before }}
/Users/jeff.geerling/Dropbox/VMs/roles/geerlingguy.passenger/tasks/main.yml:23
    repo: 'deb https://oss-binaries.phusionpassenger.com/apt/passenger {{ansible_distribution_release}} main'
```

It seems like these two rules are redundant.

**Comment by geerlingguy** [comment]
Possibly related to: https://github.com/willthames/ansible-lint/pull/384

**Comment by awcrosby** [comment]
For #384 - [TaskVariableHasSpace 205](https://github.com/willthames/ansible-lint/pull/384/files#diff-9d5dcd46349798fa432a89194e2eede4) and [VariableHasSpacesRule 208](https://github.com/willthames/ansible-lint/pull/384/files#diff-241957dc28c9896bb36f0ae634c9ae83) ... the regex in VariableHasSpacesRule catches issue this more completely, I will remove TaskVariableHasSpace
